chopper/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dev_dependencies:
   lints: ">=2.1.1 <4.0.0"
   test: ^1.24.4
   transparent_image: ^2.0.1
-  chopper_generator: ^7.1.0
+  chopper_generator: ^7.2.0
```

Comment:
Not really necessary for the release I think. But shouldn't this also use the latest (unpublished) version `7.3.0`

Reply:
It should but if I do that then the publishing mechanism won't work at all because `chopper_generator: 7.3.0` is not on pub.dev 😅

I'll bump it in a later release.